packages/jsdoc-plugins/lib/statics-augmentor/statics-augmentor.js

```diff
@@ -0,0 +1,163 @@
+/**
+ * @license Copyright (c) 2017, CKSource - Frederico Knabben. All rights reserved.
+ * Licensed under the terms of the MIT License (see LICENSE.md).
+ */
+
+'use strict';
+
+/**
+ * Manually adds doclets for inherited static members.
+ * Jsdoc doesn't support inheritance of static members which is why this plugin is needed.
+ */
+class StaticsAugmentor {
+	constructor( doclets ) {
+		this._data = doclets;
+
+		this._longnames = doclets.index.longname;
+
+		this._documented = doclets.index.documented;
+	}
+
+	/**
+	 * Main method responsible for inheriting static members.
+	 */
+	augmentStatics() {
+		const childClasses = this._getDescendants();
+
+		for ( let child of childClasses ) {
```

Comment:
I would recommend `const` instead of `let` in `for of` loop if you do not reassign the variable